portfolio/src/main/webapp/style.css

```diff
@@ -8,7 +8,7 @@ body {
     padding: 0; 
 }
 
-button {
+/* button {
```

Comment:
It's a good practice to examine the diff in your commit, before you push commit, so that you don't leave commented out code in submitted commit. Basically comment out code is a sign of bad code.